CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## 0.7.7
+- Added support for retrieving the Pubnub subscription details
```

Comment:
Currently the subscriptions are handled in Home-Assistant. Home-Assistant is making the call to pubnub with a callback to an HA method which then calls self.pubnub_update. This is probably going to change though because HA can't handle the number of connections being made to pubnub.